pages/api/admin/groups/index.js

```diff
@@ -0,0 +1,21 @@
+import { handleRequests } from "@klaudsol/commons/lib/API";
+import { withSession } from "@klaudsol/commons/lib/Session";
+import { createHash } from '@/lib/Hash';
+import { OK, NOT_FOUND } from '@klaudsol/commons/lib/HttpStatuses';
+import Groups from '@klaudsol/commons/models/Groups';
+
+export default withSession(handleRequests({ get }));
+
+async function get(req, res) {
+    const groups = await Groups.all();
```

Comment:
Kindly use assertUserCan at the beginning of all API endpoints.

https://github.com/klaudsol/klaudsol-commons/blob/main/src/lib/Permissions.js#L78